sql/core/src/test/resources/sql-tests/inputs/postgreSQL/numeric.sql

```diff
@@ -884,6 +884,12 @@ SELECT '' AS to_char_10, to_char(val, 'S0999999999999999.999999999999999'), val
 -- SELECT '' AS to_char_35, to_char('100'::numeric, 'f"ool\"999');
 -- SELECT '' AS to_char_36, to_char('100'::numeric, 'f"ool\\"999');
 
+-- [SC-131010] Add support for Snowflake built-in functions
```

Comment:
Please, don't use some internal ids like SC-131010

Reply:
Alright, will fix that.